system-test/datastore.ts

```diff
@@ -676,12 +677,18 @@ describe('Datastore', () => {
     ];
 
     before(async () => {
+      // This function is used for testing snapshot reads by getting a time and then sleeping before adding data.
```

Comment:
I believe it would be important to rephrase this comment so that it explains **why** we need the sleeping: 
is it simulating process time from datastore? is it simulating latency?